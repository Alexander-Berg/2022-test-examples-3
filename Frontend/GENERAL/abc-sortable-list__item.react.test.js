import React from 'react';
import { shallow } from 'enzyme';

import AbcSortableList__Item from 'b:abc-sortable-list e:item';

const index = 2;
const id = `test_${index}`;
const isDisabled = true;
const item = 'content';

describe('AbcSortableList__Item', () => {
    it('Should render disabled sortable list', () => {
        const itemRender = jest.fn();

        const wrapper = shallow(
            <AbcSortableList__Item
                id={id}
                index={index}
                isDisabled={isDisabled}
                item={item}
                itemRender={itemRender}
            />
        );

        expect(wrapper).toMatchSnapshot();
        wrapper.unmount();
    });
});
