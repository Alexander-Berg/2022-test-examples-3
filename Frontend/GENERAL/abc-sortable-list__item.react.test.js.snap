// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AbcSortableList__Item Should render disabled sortable list 1`] = `
<div
  className="abc-sortable-list__item"
  data-lego="react"
  style={Object {}}
>
  <Connect(Draggable)
    disableInteractiveElementBlocking={false}
    draggableId="test_2"
    index={2}
    isDragDisabled={true}
    shouldRespectForcePress={false}
  >
    <Component />
  </Connect(Draggable)>
</div>
`;
